ch01.adoc

```diff
@@ -99,7 +99,7 @@ The word "apex" refers to position of this group at the vertex of the tree of gr
 
 longitude dimension:: A dimension of a netCDF variable that has an associated longitude coordinate variable.
 
-most rapidly varying dimension:: The dimension of a multidimensional variable for which elements are adjacent in storage. When netCDF is represented in CDL, the most rapidly varying dimension is the last one e.g. **`x`** in **`float data(z,y,x)`**. C and Python NumPy use the same order as C, also called "column-major order", but Fortran uses the opposite convention, also called "row-major order", so that when netCDF variables are accessed in Fortran the most rapidly varying dimension is the first one.
+most rapidly varying dimension:: The dimension of a multidimensional variable for which elements are adjacent in storage. When a netCDF dataset is represented in CDL, the most rapidly varying dimension is the last one e.g. **`x`** in **`float data(z,y,x)`**. C and Python NumPy use the same order as CDL, also called "row-major order", while Fortran and R use the alternative order, also called "colum-major order", so that when netCDF variables are accessed in Fortran or R the most rapidly varying dimension is the first one.
```

Comment:
typo: " alternative order, also called "colum-major order" -- "colum" => "column"

Reply:
I'm not entirely sure what the standard practice is for this doc -- but for text in git, it's better to add some non-meaningful line breaks between sentences, so that the git diff doesn't put a full paragraph as a single change. I"ll let someone else comment on whether that's the standard for this doc.

Reply:
There's a style convention to start a new line in AsciiDoc for a new sentence, as Chris suggests.